js/main.js

```diff
@@ -27,6 +29,97 @@ Site = {
   },
 };
 
+Site.Sort = {
+  siteUrl: window.location.origin + window.location.pathname,
```

Comment:
its just that `,` means less js work cause thats how array -> string is formatted already